docs/cac-toml.md

```diff
@@ -0,0 +1,86 @@
+# Toml formatted CAC configuration file in a simple rust application
+This document shows how to use a TOML formatted CAC configuration file in your Rust application.
+
+## Add dependencies to your Cargo.toml
+
+```toml
+[dependencies]
+cac_toml = { git = "https://github.com/juspay/superposition.git", branch = "pest-cac-lang" }
+toml = { version = "0.8.8", features = ["preserve_order"] }
```

Comment:
we should remove the branch here, once merged to main this branch won't exist

Reply:
The default branch if omitted is `master` - so will make this `main` for now.